server/validate/email/validate.go

```diff
@@ -434,6 +440,64 @@ func (v *validator) Remove(user t.Uid, value string) error {
 	return store.Users.DelCred(user, validatorName, value)
 }
 
+// SendMail replacement
+//
+func (v *validator) Send_Mail(rcpt []string, msg []byte) error {
+
+	client, err := smtp.Dial(v.SMTPAddr + ":" + v.SMTPPort)
+	if err != nil {
+		return err
+	}
+	defer client.Close()
+	err = client.Hello(v.ServerName)
+	if err != nil {
+		return err
+	}
+	istls, _ := client.Extension("starttls")
+	if istls {
+		tlsconfig := &tls.Config{
+			InsecureSkipVerify: true,
+			ServerName:         v.SMTPAddr,
+		}
+		err = client.StartTLS(tlsconfig)
+		if err != nil {
+			return err
+		}
+	}
+	isauth, _ := client.Extension("auth")
+	if v.auth != nil && isauth {
+		err = client.Auth(v.auth)
+		if err != nil {
+			return err
+		}
+	}
+	err = client.Reset()
+	if err != nil {
+		return err
+	}
+	err = client.Mail(v.senderEmail)
+	if err != nil {
+		return err
+	}
+	for _, to := range rcpt {
+		err = client.Rcpt(to)
```

Comment:
Please check for presence of `"\n\r"` in `to` (or just strip them).

Reply:
fixed.